script/heroku-postbuild.sh

```diff
@@ -11,4 +11,4 @@ git clone --depth=1 https://github.com/department-of-veterans-affairs/vagov-cont
 cd ${vagov_apps_dir}
 
 # build the site as usual
-npm run build --  --entry static-pages --content-directory ${vagov_content_dir}/pages
+npm run build --  --entry static-pages,caregivers --content-directory ${vagov_content_dir}/pages
```

Comment:
Need to revert after PR is reviewed

Reply:
AFAIK the heroku instances don't run the js apps.

Reply:
@rianfowler  They don't? I swear I used them in the past before...

Reply:
the review instances get the js app but I'm pretty sure the heroku ones only deploy the va-content repo stuff. I don't think there's any webpack stuff happening there. guess we can see- the heroku build is stuck in pending status

Reply:
In my experience the apps work just the backend calls do not. Have used the Heroku instance to demo to stakeholders in the past, but its possible things have changed with the instance updates 🤷‍♂

Reply:
https://vetsgov-pr-11852.herokuapp.com/caregivers/

Reply:
Since I am a dummy head, got excited, and forgot to remove this line of code -_-
https://github.com/department-of-veterans-affairs/vets-website/pull/11943

![](https://gph.is/1fFAj2t)